virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -586,3 +586,30 @@ def fill_null_in_vm(vm, target, size_value=500):
             raise exceptions.TestError("Error happened when executing command:\n%s" % cmd)
     except Exception as e:
         raise exceptions.TestError(str(e))
+
+
+def check_virtual_disk_io(vm, partition):
+    """
+    Check if the disk partition in vm can be normally used.
+
+    :param vm: Vm instance
+    :param partition: the disk partition in vm to be checked.
+    :return: if the disk can be used, return True
+    """
+    session = None
+    try:
+        session = vm.wait_for_login()
```

Comment:
Is that OK if check vm state and start vm before login,  because start vm is a mandatory step before this function ?

Reply:
Here vm.wait_for_login() happen because we need get one session to access VM internal. 
Checking VM state is more specific to test case,that often happen in your tp-libvirt test case code.